src/Orc.Controls/Controls/Callout/Models/CalloutManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Orc.Controls
+{
+    using System.Collections.Generic;
+
+    public class CalloutManager : ICalloutManager
+    {
+        public CalloutManager()
+        {
+        }
+
+        public IList<CalloutViewModel> Callouts { get; set; } = new List<CalloutViewModel>();
+
+        public void Register(CalloutViewModel calloutViewModel, Callout callout)
+        {
+            if (calloutViewModel != null && callout != null)
```

Comment:
please use "is not null". But even better, this method should throw when passing in null. So use:

Argument.IsNotNull(() => calloutViewModel);
Argument.IsNotNull(() => callout);